cmake/buildTargets.cmake

```diff
@@ -9,10 +9,12 @@ if (VINECOPULIB_SHARED_LIB)
 else()
     add_library(vinecopulib INTERFACE)
     target_include_directories(vinecopulib INTERFACE
-            $<BUILD_INTERFACE:${vinecopulib_includes}>
-            $<INSTALL_INTERFACE:include/vinecopulib>)
+            $<BUILD_INTERFACE:${vinecopulib_includes}>)
 endif()
 
+target_include_directories (vinecopulib INTERFACE $<INSTALL_INTERFACE:include>)
+target_link_libraries(vinecopulib INTERFACE Eigen3::Eigen wdm)
```

Comment:
Why is the link to boost not needed then?

Reply:
indeed lets add boost too